addons/kafka/templates/cmpd-broker-2.8.1.yaml

```diff
@@ -0,0 +1,201 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  name: "{{ include "kafka-broker.componentDefName" . }}-2.8.1"
```

Comment:
refact all version string to 2.8.2

Reply:
The 2.8.2 image is still being tested. I'll push it again once there are no issues.